.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+
+**TIP:** If you are not sure where to start contributing, you can check GitHub issues: [help wanted](https://github.com/magento/devdocs/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) and [good first issue](https://github.com/magento/devdocs/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
+
+## Contribution guidelines
+We use [Markdown](http://daringfireball.net/projects/markdown/) to write our documentation, which is a simple markup language that we convert to HTML using [Kramdown](http://kramdown.gettalong.org/syntax.html).
+
+You can update, or add content to, existing topics in their respective versioned directories (for 2.0, 2.1, 2.2, and onward).
+
+The following guidelines may answer most of your questions and help you get started:
+
+1.  Check [existing pull requests](https://GitHub.com/magento/devdocs/pulls) and make sure you are not duplicating work!
+
+1.  For large contributions or changes that include multiple files, [open an issue](#report-an-issue) and discuss it with us first. This may further prevent duplicate or unnecessary effort.
+
+1.  Familiarize yourself with the existing documentation. Look through and search the guides to decide where to add your topics.
+
+    -   The DevDocs team can find the best home for your new topics and add it to the navigation.
+    -   If a topic has a symlink, you can remove it with git commands and add a new file. Copy and paste a previous version of the topic to get started.
+
+1.  Focus on the content and on creating useful information for your fellow Magento developers and community members. Don't forget to review your work for typos, formatting errors, or sentences that need clarifying before opening a pull request.
+
+1.  Use the following guidelines to help you with the writing process:
+
+    -   Define the goal of your topic. What exactly do you want to teach the reader?
+    -   Make the title of your topic reflect the content.
+    -   Keep your sentences concise and try to separate conceptual information from procedural steps.
+    -   Remember to use active voice (not passive), write in the present tense, and use a friendly tone in second person. For example, _"You can now view the output..."_.
+    -   Use notes to alert readers about important details.
+    -   Use cross-references to other topics if appropriate. We can help you with the syntax if it is not clear. The template provides an example you can use.
+
+## Preview HTML locally
+
+To preview your changes in HTML output, follow the instructions in the [README](https://GitHub.com/magento/devdocs/blob/develop/README.md) to build the devdocs site locally using Jekyll.
+
+## Templates
+
+Edit and add content to existing pages. For new topics, we provide templates to help get you started:
+
+* [General topic template](https://devdocs.magento.com/guides/v2.2/contributor-guide/templates/basic_template.html): This is a template for writing any topic with example formats and styles.
```

Comment:
Since the intro sentence uses a : maybe use hyphens to separate the description in each bullet point.